src/pycatcher/catch.py

```diff
@@ -479,7 +486,7 @@ def detect_outliers_moving_average(df: pd.DataFrame) -> str:
     df1['above_threshold'] = df_pandas.iloc[:, -1] > (df1['moving_average'] + threshold)
     df1['below_threshold'] = df_pandas.iloc[:, -1] < (df1['moving_average'] - threshold)
 
-    outliers = df1[(df1['above_threshold'] == True) | (df1['below_threshold'] == True)].dropna()
+    outliers = df1[(df1['above_threshold']) | (df1['below_threshold'])].dropna()
```

Comment:
To adhere to PEP 8 standards and resolve the E712 error (comparison to True should be avoided)